### ISSUE TITLE
Request to add Ŭ and ŭ

### ISSUE THREAD

**Issue by fortran77** [google/extra-keyboards-for-chrome-os#24]
Currently Esperanto users are making do with makeshift solutions for entering the Esperanto-specific characters Ĉĉ Ĥĥ Ĵĵ Ŝŝ Ŭŭ.

As it so happens, the [ComposeKey](https://chrome.google.com/webstore/detail/composekey/iijdllfdmhbmlmnbcohgbfagfibpbgba) extension includes Ĉĉ Ĥĥ Ĵĵ Ŝŝ already.

**The only one missing is Ŭŭ.**

If Ŭŭ were also added, then this same extension would work nicely for Esperanto too, simplifying life for everybody.

**Comment by LaPingvino** [comment]
US INTL already has full support, to type the ŭ and Ŭ there, type ˘ first by typing shift-altgr-9, then type u/U. The Dvorak International keyboard provides the same with a Dvorak layout.

**Comment by fortran77** [comment]
Thanks very much for the follow-up.

At some point after I opened this issue, Ŭŭ did become available in the [ComposeKey](https://chrome.google.com/webstore/detail/composekey/iijdllfdmhbmlmnbcohgbfagfibpbgba) extension as right-Alt–b–U and right-Alt–b–u. I'm not sure when it got added. I suspect the extension as available in the Chrome Web Store is not exactly in sync with the source code here in this repository.

Let me also take this opportunity to put the following down where it can be found in the future by people looking for ideas about how to improve both ComposeKey and the Chrome browser.

While looking for Ŭŭ some months ago I searched the entire Internet and found no authoritative source that gave me a list of all the characters that could be typed on the Chrome OS US INTL keyboard. Many different websites document this in bits and pieces, but they usually describe several different international keyboards which vary slightly from one another. Whereas the ComposeKey extension comes with an actual README file that lists every single character that is available. For this reason the ComposeKey extension is much more useful, as one can look up the documentation in a single place.

The ComposeKey extension is also much more convenient than the US INTL keyboard because the right-Alt key remains out of the way and does not interfere with normal typing. Whereas in the case of the US INTL keyboard it becomes inconvenient to use many special characters to stand for themselves, as they require one to type a space after them.

For the above reasons, Google should make ComposeKey more general (by adding additional most-wanted international characters), and support it as a Chrome browser extension on all platforms, not just in Chrome OS.
